src/support/hazard.c

```diff
@@ -174,7 +174,7 @@ __wt_hazard_set_func(WT_SESSION_IMPL *session, WT_REF *ref, bool *busyp
      * return the page to use, no harm done, if it doesn't, it will go ahead and complete the
```

Comment:
Up on line 146 there are two occurrence of "flush" that can become either write or publish

Reply:
Moved into [WT-10967](https://jira.mongodb.org/browse/WT-10967) 👍